Sec role doesn't exist in CM; make it clearer. :-) (#1152)
MicrosoftDocs/SCCMdocs: sccm/core/servers/manage/cmpivot.md

- Upgrade the target devices to the latest version of the Configuration Manager client.  

> - The Configuration Manager administrator needs the **Read** permission on the **SMS Scripts** object, the **Run Scripts** permission on the **Collection**, and the default scope. The **Scripts Runner** role has these permissions, which is not created by default withn ConfigMgr. For more information on creating this ConfigMgr Security Role, please see [Security roles for scripts](/sccm/apps/deploy-use/create-deploy-scripts#bkmk_ScriptRoles).  

Reviewer: A few editorial suggestions:

```suggestion
- The Configuration Manager administrator needs the **Read** permission on the **SMS Scripts** object, the **Run Scripts** permission on the **Collection**, and the default scope. The **Scripts Runner** role has these permissions, which isn't created by default. For more information on creating this custom security role, see [Security roles for scripts](/sccm/apps/deploy-use/create-deploy-scripts#bkmk_ScriptRoles).  
```
